configure.ac

```diff
@@ -357,20 +377,24 @@ AC_SUBST(SSL_LIBS)
 
 have_winsock=disabled
 WINSOCK_LIBS=
-AC_ARG_WITH([winsock],
-            [AS_HELP_STRING([--with-winsock],
-                            [support Windows Sockets API @<:@default=check@:>@])],
-            [],
-            [with_winsock=check])
 
-if test "x$with_winsock" != "xno"
+if test "x$with_cygwin" = "xno"
```

Comment:
Winsock support will be detected on Cygwin, but will induce compilation errors if an attempt is made to actually build with it.